### ISSUE TITLE
No Data Loading

### ISSUE THREAD

**Issue by ewarr1sc** [EMU-CFE/CanvasDataViewer#25]
We are running without any errors however, data is not loading into the tables once created. Is anyone having similar issues? apologies i can't be more specific

**Comment by wjones20** [comment]
Yes, I'm noticing the same thing.  Just happened in the last two days,
because I was looking at it the day before.  I'm going to write to Canvas
and ask if something changed.
*********************************************************
Bill Jones, MA, MS      Director, Instr Tech & Info Mngmt
Center for E-Learning                  ph: (734) 487-9043
Eastern Michigan University           fax: (734) 487-6695
Ypsilanti, MI 48197             email: wjones20@emich.edu
*********************************************************

**Comment by ewarr1sc** [comment]
> Yes, I'm noticing the same thing. Just happened in the last two days, because I was looking at it the day before. I'm going to write to Canvas and ask if something changed. ********************************************************* Bill Jones, MA, MS Director, Instr Tech & Info Mngmt Center for E-Learning ph: [(734) 487-9043](tel:(734) 487-9043) Eastern Michigan University fax: [(734) 487-6695](tel:(734) 487-6695) Ypsilanti, MI 48197 email: wjones20@emich.edu *********************************************************
> […](#)

Have you had any of your data pull back through? Ours is back, I reran the scheduled jobs and the majority seems to be there. I need to check the numbers as Canvas is showing different learner numbers enrolled on certain courses when compared to a simple query ran through SQL
